apps/theming/css/default.css

```
@@ -50,7 +50,7 @@
   --font-face: system-ui, -apple-system, 'Segoe UI', Roboto, Oxygen-Sans, Cantarell, Ubuntu, 'Helvetica Neue', 'Noto Sans', 'Liberation Sans', Arial, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol', 'Noto Color Emoji';
   --default-font-size: 15px;
```

Maybe add a small font size, as you're touching it?
```suggestion
  --default-font-size: 15px;
  --font-size-small: 13px;
```

---

Was going to do it in a separate pr, but ok

---

Agree of creating PR to no mix creating a new variable (feature) and fixing line-height (design change).